setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup(
     name="cidc_cli",
-    version="0.2.0",
+    version="0.2.1",
```

Comment:
I suggest we keep version of all components somewhat synced. Api is already 0.3.0, and I suggested 0.4.0 there so 
```suggestion
    version="0.4.0",
```